apps/sms/views/inbox/js/inbox.js

```diff
@@ -974,7 +974,7 @@ var InboxView = {
     }
 
     if (thread.isDraft) {
-      this.removeThread(thread.id);
+      this.deleteThread(thread.id);
```

Comment:
> BTW maybe we can rename the removeThread as well because it's misleading, removeThreadDom or something could be more clear.

Yep, I thought about the same! Will rename